websites/P/PreMiD/dist/metadata.json

```diff
@@ -19,6 +19,10 @@
     {
       "name": "Callum",
       "id": "506899274748133376"
+    },
+    {
+      "name": "Slowftw",
+      "id": "325046217904226306"
     }
```

Comment:
Unfortunately, you will have to remove this from the metadata file for the following reason:
```
If you make modifications to a Presence and change at least a QUARTER of the Presence's codebase, you are allowed to add yourself as a contributor. Contact a Presence Verifier for more information about this subject.
```

Reply:
o ok
I forgot to take that into consideration